src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
```

Comment:
Fields 'color' and 'number' should have access modifiers to specify their visibility explicitly. In Java, it's considered good practice to make class fields private to follow the encapsulation principle.